### DISCUSSION TITLE
(Solved) HTTP Error 500 after update from version 2.1.9 to 2.2.4

### DISCUSSION THREAD

**Discussion by joaobrazuna** [Leantime/leantime#831] [Q&A] [answered]
Hi there,

After updating leantime from version v2.1.9 to v2.2.4 I receive the following error:
HTTP Error 500

After activating $Degug=1 in config, the error message changed to:

Parse error: syntax error, unexpected '|', expecting ';' or '{' in /var/www/html/leantime/src/domain/reports/services/class.reports.php on line 121

After reading some discussions here, and looking to the "parse error" I realized that leantime now needs PHP 8. (I was still using version 7)
I did the upgrade to version 8 and now it's working fine :-)

**Comment by marcelfolaron** [comment, marked as answer]
> Hi there,
> 
> After updating leantime from version v2.1.9 to v2.2.4 I receive the following error: HTTP Error 500
> 
> After activating $Degug=1 in config, the error message changed to:
> 
> Parse error: syntax error, unexpected '|', expecting ';' or '{' in /var/www/html/leantime/src/domain/reports/services/class.reports.php on line 121
> 
> After reading some discussions here, and looking to the "parse error" I realized that leantime now needs PHP 8. (I was still using version 7) I did the upgrade to version 8 and now it's working fine :-)
